libraries/disp3D/disp3D.pro

```diff
@@ -91,7 +91,6 @@ SOURCES += \
     engine/model/items/freesurfer/fsannotationtreeitem.cpp \
     engine/model/items/hemisphere/hemispheretreeitem.cpp \
     engine/model/items/sourcespace/sourcespacetreeitem.cpp \
-    engine/model/items/sourcedata/mneestimatetreeitem.cpp \
```

Comment:
Pls keep order of the files in tact. Just rename the .cpp and .h file names in place here.